app/components/live-participants.js

```diff
@@ -8,4 +8,15 @@ export default class LiveParticipantsComponent extends Component {
   get liveService() {
     return getOwner(this).lookup('service:live');
   }
+
+  get participantsCount() {
+    let count = 0;
+    let { peers, role } = this.args;
+    for (let i = 0; i < peers.length; i++) {
+      if (peers[i].roleName === role) {
+        count++;
+      }
+    }
+    return count;
+  }
```

Comment:
Can we simply filter with the help of role and do .length to get the peers count?

Reply:
done